clarity.d.ts

```diff
@@ -25,6 +25,10 @@ interface IConfig {
   // Send back instrumentation data, if set to true
   instrument?: boolean;
 
+  // Pointer to the function which would be responsible for sending the data
+  // If left unspecified, raw payloads will be uploaded to the uploadUrl endpoint
+  sendCallback?: (payload: string, onSuccess?: (status: number) => void, onFailure?: (status: number) => void) => void;
```

Comment:
Should you add some sort of unique identifier for tracking the request? Otherwise how do you know which failed (unless you use a unique callback per invocation).